### PR TITLE
Use Kysely for PLC DB

### PR DESCRIPTION
**Pull request by dholms**
This started out as just checking out Kysely to get a feel for it in a smaller context. 
It's much better than TypeORM. 
- Better typing in queries
- Transactions that _actually_ work & don't require a mutex
- Dismisses the whole ORM abstraction, which I think is more honest & probably better tooling for server admins

We're planning to switch over TypeORM to Kysely on the server, so this is now an _actual_ PR & not just a testing ground 😛

### CODE REVIEW

File: packages/plc/src/server/db.ts [hunk 1]
```diff
@@ -1,44 +1,59 @@
...
 import * as t from '../lib/types'
 import { ServerError } from './error'
 
-export class Database {
-  db: SafeDB
+interface OperationsTable {
+  did: string
+  operation: string
+  cid: string
+  nullified: 0 | 1
```
Comment (dholms) [inline packages/plc/src/server/db.ts:13]: better-sqlite3 forces you to be honest about sqlite values & doesn't let you use booleans 😛

File: packages/plc/src/server/db.ts [hunk 2]
```diff
@@ -1,44 +1,61 @@
...
```
Comment (devinivy) [inline packages/plc/src/server/db.ts:13]: If we want to wrangle this quirk using types, we may be able to use an enum.  E.g. something along the lines of:
```ts
enum DbBool {
  true = 1,
  false = 0,
}
```

**Review by devinivy — APPROVED** [review]
Big yup! 👍

File: packages/plc/src/server/db.ts (hunk 2 above)
Comment (dholms) [reply packages/plc/src/server/db.ts:13]: im not too worried about it just yet. if we're feeling friction from it in the future, then totally 👌

**Pull request merged**